### ISSUE TITLE
Looking for ways to expand!

### ISSUE THREAD

**Issue by Piyusharora2003** [Satakshijain/CodingHustlers#41]
So basically right now the project do nothing at all just a home page with some nav links .

But what can we do 

- Make it next w3School
- Make it next GeeksForGeeks 
- Build an online learning system (I did had a working prototype for the same , i build for[ json schema](https://a-tour-to-json-schema.vercel.app/) (Postman) 
- Provide blogs for the roadmaps

**Comment by IP80808080** [comment]
👋 Hello! I'd be happy to help with this issue. Could you please assign it to me? Additionally, could you please label it for the appropriate level? Thank you! 🚀

**Comment by Piyusharora2003** [comment]
@IP80808080 Did you even read what i wrote ?

**Comment by IP80808080** [comment]
You want to expand the project right? 
like these e.g 
Make it next w3School
Make it next GeeksForGeeks
right ?
If I got something miss understanding ping me

**Comment by Piyusharora2003** [comment]
yes, but how can i assign it to someone , firstly i am just a contributor like you . secondly i believe it would rather be a discussion than an issue
